webapp/handlers.py

```diff
@@ -114,6 +120,21 @@ def clear_trailing():
 
     @app.before_request
     def prometheus_metrics():
+        # Accept-encoding counter
+        # ===
+        agent_string = flask.request.headers.get("User-Agent")
+
+        # Exclude k8s probes, which happen behind the cache
+        if not agent_string.startswith("kube-probe"):
```

Comment:
sorry another one. We are also getting this user agent from behind the cache: `Prometheus/2.9.2`

:sweat: